orator/commands/command.py

```diff
@@ -77,7 +77,7 @@ def _check_config(self):
         """
         current_path = os.path.relpath(os.getcwd())
 
-        accepted_files = ['orator.yml', 'orator.py']
+        accepted_files = ['orator.yml', 'orator.py', 'config.py', 'db.py', 'config.json', 'db.json']
```

Comment:
Regarding JSON support, I will think about it.

I made the config YAML or Python only because that's what I use most of the time but JSON might be a nice addition.